Source/CombinePromise.swift

```diff
@@ -52,6 +59,12 @@ public final class CombinePromise: Promise<[Any]> {
         incrementPromise()
     }
     
+    /**
+    Initializer for `CombinedPromise` requiring list of other `Promisable`s to wait on before being fired. If one or more of these `Promisable` values are rejected this `CombinedPromise` is also rejected. The protocol `Promisable` instead of the `Promise` class is used here to allow `Promise` objects with mixed associated values to be specified.
```

Comment:
I've seen long descriptions like this broken up on multiple lines. Might be nicer to read for people who don't have wrapping turned on.